docs/_plugins/markers-renderers/leaflet-awesomemarkers.md

```diff
@@ -4,7 +4,7 @@ category: markers-renderers
 repo: https://github.com/lennardv2/Leaflet.awesome-markers
 author: Lennard Voogdt
 author-url: https://lennardvoogdt.nl/
-demo: 
+demo: https://github.com/lennardv2/Leaflet.awesome-markers#screenshots
```

Comment:
Not sure whether a screenshot section of a readme (or a readme alone) counts as a demo. There are some like this.

Reply:
That specific example is borderline since the README provides an unstable JSFiddle URL under the screenshot. I think it's hard to be maintained in Leaflet documentation.